bin/bump-versions

```diff
@@ -0,0 +1,174 @@
+#! /bin/python3
+
+import os
+from os.path import join, dirname, abspath
+
+# Usage: see below. 
+# 
+# @author Steven Wong <housengw@berkeley.edu>
+def usage(): 
+    return  '''bump-versions.py [-h] [-p PROP] [-t TARGET] [-a]
+
+            update the version number in config.properties. 
+            Then, run `python3 bump-versions.py --all` to update for all targets (maven and manifest files).
```

Comment:
```suggestion
            Then, run `python3 bump-versions --all` to update for all targets (maven and manifest files).
```

Reply:
Looks like the `bump-versions` file doesn't have a .py extension. I think it might make more sense to add it though.

Reply:
It's an executable, so I think it shouldn't have an extension? IDK, are there guidelines for this anyway?

Reply:
https://stackoverflow.com/questions/47360213/should-executable-python-script-be-named-with-py/47360365

Reply:
We don't have an extension for `lfc`, obviously. I think it would be silly to have that. Then why have it for the version bump script?

Reply:
It's not meant to be used as a module, anyway...

Reply:
I'm going with consistency for now; since it's in a `bin` directory and is executable, it should have no extension. We could later move it to a `scripts` directory, make it non-executable and give it a `.py` extension if that is preferable.

Reply:
My understanding is that having the `.py` makes it more portable.
I agree that this script might not belong in `bin` though.